ports/servoshell/minibrowser.rs

```diff
@@ -295,8 +295,8 @@ impl Minibrowser {
         }
     }
 
-    /// Updates the location field from the given [BrowserManager], unless the user has started
-    /// editing it without clicking Go, returning true iff it has changed (needing an egui update).
+    /// Updates the location field from the given [Minibrowser], unless the user has started
+    /// editing it without clicking Go, returning true if it has changed (needing an egui update).
```

Comment:
this `iff` is correct, we don't need to change it to `if`. `iff` is shortened form of `if and only if`